prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -8,11 +8,22 @@ presubmits: # runs on PRs
       context: prow/kyma/kyma-integration
       skip_report: true # from documentation: SkipReport skips commenting and setting status on GitHub.
       max_concurrency: 10
-      labels:
+      labels: &kyma-integration-labels
```

Comment:
In fact, it didn't simplify that YAML. Please take a look at @aszecowka proposal in this PR: https://github.com/kyma-project/test-infra/pull/91/files#diff-aa49fee1a36403268cdeb2786a4c6fb6

I know it is not a scope of this task, so you can revert anchors and create followup task for simplification.

Reply:
I reverted the changes from this PR and created this follow-up task: https://github.com/kyma-project/test-infra/issues/102